codalab/rest/util.py

```diff
@@ -95,6 +98,25 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+# For non-REST services, should call with client=CodaLabManager
+@local_bundle_client_compatible
+def notify_admin(local, request, message):
+    if 'admin_email' in local.config['server']:
```

Comment:
Just updated deployment:
https://github.com/codalab/codalab-worksheets/pull/206

(and manually on NLP instance)